src/lib/y2packager/clients/inst_productsources.rb

```diff
@@ -753,15 +745,11 @@ def ReadDialog
       @language_long = GetCurrentLang() if !Stage.initial
 
       # fallback if no LANG variable set
-      if @language_long.nil? || @language_long == ""
-        @language_long = Language.language
-      end
+      @language_long = Language.language if @language_long.nil? || @language_long == ""
 
       # de_DE.UTF-8 --> de_DE
       dot_pos = Builtins.search(@language_long, ".")
```

Comment:
@mvidner recently created a class that handled locale names, maybe we should use it here...

Reply:
yep, but not goal of this PR.